libminifi/include/capi/Plan.h

```diff
@@ -16,6 +16,8 @@
  * limitations under the License.
  */
 
+#pragma once
```

Comment:
The more I think about this for a public, open source project, we should probably avoid pragma once given that it's implementation defined and not language spec. We don't control all compiler variants, especially for what will be this portion of the API. I realize it's nearly "universally supported" but it seems like an unnecessary risk, albeit small.

Reply:
As it's Plan.h, this will surely be compiled with C++ compilers, which properly support according to my experience. I don't plan to add it to C API.

Reply:
We should keep to the standards.

Reply:
Removed.